src/index.ts

```diff
@@ -857,9 +859,12 @@ export class Jack<C extends ConfigSet = {}> {
           { name: field, found: value }
         : undefined
       if (cause) {
-        throw new Error(`Invalid config value for ${field}: ${value}`, {
-          cause,
-        })
+        throw new Error(
+          `Invalid config value for ${field}: ${JSON.stringify(value)}`,
```

Comment:
This was also found via linting. The change helps distinguish between primitive and non-primitive values in error messages:

```js
import { jack } from "jackspeak";

const j = jack()
  .optList({ xlist: { validate: () => false } })
  .opt({ x: { validate: () => false } });

const validate = (o) => {
  try {
    j.validate(o);
  } catch (e) {
    console.log(e.message);
  }
};

validate({ xlist: ["a", "b", "c"] });
validate({ x: "a,b,c" });
```

before**
```
❯ node index.mjs
Invalid config value for xlist: a,b,c
Invalid config value for x: a,b,c
```

after
```
❯ node index.mjs
Invalid config value for xlist: ["a","b","c"]
Invalid config value for x: "a,b,c"
```